code/go/0chain.net/blobbercore/handler/grpc_helpers.go

```diff
@@ -18,14 +18,14 @@ type gRPCHeaderMetadata struct {
 	ClientSignature string
 }
 
-
 func registerGRPCServices(r *mux.Router, server *grpc.Server) {
 	blobberService := newGRPCBlobberService()
 	r.Use(Middleware2("ds"))
 	grpcGatewayHandler := runtime.NewServeMux(
-								runtime.WithIncomingHeaderMatcher(CustomMatcher),
-							)
+		runtime.WithIncomingHeaderMatcher(CustomMatcher),
+	)
 
+	_ = grpcGatewayHandler.HandlePath("POST", `/v1/file/upload/{allocation}`, uploadHandler)
```

Comment:
Can we not re-use the already implemented HTTP handler in the `handler.go` file?

Reply:
Yes, we can rewrite StorageHandler Methods for it